app/api/server/v1/out-of-office.ts

```diff
@@ -0,0 +1,69 @@
+import { Meteor } from 'meteor/meteor';
+import { check, Match } from 'meteor/check';
+
+import { API } from '../api';
+import { settings } from '../../../settings/server';
+import { updateOutOfOffice } from '../../../out-of-office/server';
+import { OutOfOfficeUsers } from '../../../models/server';
+
+API.v1.addRoute(
+	'outOfOffice.toggle',
```

Comment:
this might not be very clear, but we have a pattern to define REST endpoint names as you can see at https://handbook.rocket.chat/product/development/development-guidelines#standards

it says we use a dashed namespace, so in this case the correct would be `out-of-office.toggle`

Reply:
Renamed the toggle endpoint to `out-of-office.toggle`!